README.md

```diff
@@ -20,21 +20,21 @@ This is achieved by creating YAML files that contain a pre-determined data schem
 
-As part of the VXLAN as Code service from Cisco, you will also be able to utilize the semantic validation to make sure that the data model matches the intended expected values. This is a powerful feature that allows you to ensure that the data model is correct before it is deployed to the network. Also, part of the validate role is the ability to create rules that can be used to avoid operators from making specific configurations that are not allowed in the network. These can be as simple as ensuring naming convention to more complex rules for interconnectivity that would need to be avoided. These would be coded in python and can be constructed as part of the Services as Code offer.
+As part of the VXLAN as Code service offer from Cisco, you will also be able to utilize the semantic validation to make sure that the data model matches the intended expected values. This is a powerful feature that allows you to ensure that the data model is correct before it is deployed to the network. Also, part of the validate role is the ability to create rules that can be used to avoid operators from making specific configurations that are not allowed in the network. These can be as simple as ensuring naming convention to more complex rules for interconnectivity that would need to be avoided. These would be coded in python and can be constructed as part of the Services as Code offer.
 
 ### Create role
 
 Role: [cisco.nac_dc_vxlan.dtc.create](https://github.com/netascode/ansible-dc-vxlan/blob/develop/roles/dtc/create/README.md)
 
-This role is going to create all the templates and variable parameters that are going to be used in the deployment of the VXLAN fabric. This role converts the data model into the proper templates that are required by the Ansible module to be able to communicate with the NDFC controller.
+This role is going to use a templates and variable parameters from the data model to create and stage all the configuration for your VXLAN fabric. This role uses the data model in the proper templates to render the expected data that is required by the NDFC Ansible modules.
 
 ### Deploy role
 
 Role: [cisco.nac_dc_vxlan.dtc.deploy](https://github.com/netascode/ansible-dc-vxlan/blob/develop/roles/dtc/deploy/README.md)
 
-The deploy role is going to deploy those changes to the NDFC controller. This role is going to take the templates and variable parameters that were created in the `create` role and deploy them to the NDFC controller. This is the role that is going to make the changes in the NDFC controller.
+The deploy role is going to recalculate and deploy the staged changes from the `create` role to your NDFC controller.
```

Comment:
No, the deploy roles triggers NDFC to recalculate and deploy the configuration (which already has been applied to NDFC by the create role) to the switches.

Reply:
Again, staged in fact means staging the configuration within NDFC, so I believe we're saying the same thing, no?

Reply:
Currently it says "deploy to NDFC", but it's actually "deploy from NDFC to the switches". Therefore I thought it meant creating the configuration in NDFC what is done in the create role. I'd suggest to rephrase it like "The deploy role triggers NDFC to recalculate and deploy the configuration to the switches."

Reply:
The documentation in our example repo is clearer so I propose we use that here as well.